examples/custom-metrics-api.jsonnet

```diff
@@ -0,0 +1,11 @@
+local kp =
+  (import 'kube-prometheus/kube-prometheus.libsonnet') +
+  (import 'kube-prometheus/kube-prometheus-custom-metrics.libsonnet') +
+  {
+    _config+:: {
+      namespace: 'monitoring',
+    },
+  };
+
+{ ['setup/0namespace-' + name]: kp.kubePrometheus[name] for name in std.objectFields(kp.kubePrometheus) } +
```

Comment:
Technically you would also need a prometheus, the operator etc. Not sure if all of this should also be generated (it wasn't before)